matching-notification-ui/src/main/resources/MatchingNotification/MatchesTable.xml

```diff
@@ -127,20 +128,35 @@ $xwiki.jsx.use('PhenoTips.Widgets')##
               &lt;/div&gt;
             &lt;/ul&gt;#end
             &lt;span id="advanced-filters-header" class='buttonwrapper'&gt;
-              &lt;a class='button toggle-filters #if("${doc.space}" != "data")secondary-filters'#else' id="toggle-filters-button"#end href='#' &gt;{{icon name='filter' /}}&lt;span class='label'&gt;$services.localization.render('phenotips.myMatches.filterButton')&lt;/span&gt; #if("${doc.space}" == "data"){{icon name='angle-double-left' cssClass='collapse-marker' /}}{{icon name='angle-double-down' cssClass='expand-marker' /}}#end&lt;/a&gt;
+              &lt;a class='button toggle-filters #if("${doc.space}" != "data")secondary-filters'#else' id="toggle-filters-button"#end href='#' &gt;&lt;span class="fa fa-filter"&gt; &lt;/span&gt;&lt;span class='label'&gt;$services.localization.render('phenotips.myMatches.filterButton')&lt;/span&gt; #if("${doc.space}" == "data")&lt;span class="fa fa-angle-double-left  collapse-marker"&gt; &lt;/span&gt;&lt;span class="fa fa-angle-double-down  expand-marker"&gt; &lt;/span&gt;#end&lt;/a&gt;
             &lt;/span&gt;
             &lt;ul class="filters xform"&gt;
               &lt;li id="checkbox-server-filters"&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.serverIdCheckbox.label'))&lt;br&gt;
-                &lt;label&gt;&lt;input name="checkbox-server-id-filter" type="checkbox" value="local" checked &gt;
-                  $escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.matchesSeletFilter.local'))&lt;span class="matches-count"&gt;&lt;/span&gt;&lt;br&gt;
+                &lt;label&gt;&lt;input name="checkbox-server-id-filter" type="checkbox" value="local" checked &gt;##
+                  $escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.matchesSeletFilter.local'))&lt;span class="matches-count"&gt;&lt;/span&gt;##
+                  #set ($disclaimer = "$!xwiki.getDocument('PhenomeCentral.ServiceDisclaimer').getRenderedContent()")##
+                  &lt;input type="hidden" class="disclaimer" value="${escapetool.xml($disclaimer)}"&gt;##
```

Comment:
The hidden inputs and the info icon only need to be generated if the disclaimer is not empty. Same comment for all remote servers.